psql2mysql.py

```diff
@@ -72,7 +76,7 @@ def scanTablefor4ByteUtf8Char(self, table):
             for col in stringColumns:
                 if type(row[col]) is not unicode:
```

Comment:
I'm stuck in this part, because in python3 all strings are unicode

Reply:
Using `if not isinstance(row[col], six.string_types)` should work here.